packages/internal/src/files.ts

```diff
@@ -99,15 +99,15 @@ export const findPrerenderedHtml = (cwd = getPaths().web.dist) =>
   fg.sync('**/*.html', { cwd, ignore: ['200.html', '404.html'] })
 
 export const isCellFile = (p: string) => {
-  const { dir, name } = path.parse(p)
+  const { name } = path.parse(p)
 
   // If the path isn't on the web side it cannot be a cell
   if (!isFileInsideFolder(p, getPaths().web.src)) {
     return false
   }
 
-  // A Cell must be a directory named module.
-  if (!dir.endsWith(name)) {
+  // A cell must end with "Cell.{jsx,js,tsx}".
+  if (!name.endsWith('Cell')) {
```

Comment:
Why this change? 

I'm wondering if this has a downstream impact elsewhere?

Reply:
We used to have the name check out in the watcher function, but I thought it was such a fundamental check that I wanted it to be part of the `isCellFile` check itself. So that we always check this when checking if a file is a Cell.

The reason I removed the "directory named module"-check is because I think it's wrong. In one of my projects I have a "child cell" (for a waterfall kind of setup), and I chose to put that Cell in the same directory as the parent Cell. Works great.

This also aligns with the comment in `RWCell.ts` in the structure package:
```
  /**
   * A "Cell" is a component that ends in `Cell.{js, jsx, tsx}`, has no
   * default export AND exports `QUERY`
   **/
 ```
And it aligns with our public facing docs
> Redwood looks for all files ending in "Cell" (so if you want your component to be a Cell, its filename does have to end in "Cell"), but if the file 1) doesn't export a const named QUERY and 2) has a default export, then it'll be skipped.
https://redwoodjs.com/docs/cells#how-does-redwood-know-a-cell-is-a-cell

As far as downstream impact goes the `isCellFile` function is used (by extension) in our type generator and directly in the watcher. So I think we're good on that front.

Looking at all this again I think maybe we should also remove the check for a `Success` export below to further align this implementation with the other code and docs

Reply:
Would the generated mirror types be correct for a non-directory-import Cell? If so, happy with it and makes sense.

Reply:
The project I'm referring to is actually a JS project. So I only know that the basic Cell functionality works. Don't know about the types. I'll make sure to test it manually.